main.js

```diff
@@ -75,6 +77,9 @@ menubarApp.on('ready', () => {
       }
     }
   });
+  ipcMain.on('get-platform', (event) => {
```

Comment:
We should switch to this approach for all usage of remote. See https://github.com/gitify-app/gitify/issues/650